.github/workflows/bindings_nodejs.yml

```diff
@@ -36,42 +36,6 @@ on:
   workflow_dispatch:
 
 jobs:
-  test:
-    runs-on: ubuntu-latest
-
-    # Notes: this defaults only apply on run tasks.
-    defaults:
-      run:
-        working-directory: "bindings/nodejs"
-
-    steps:
-      - uses: actions/checkout@v4
-      - name: Setup Rust toolchain
-        uses: ./.github/actions/setup
-      - name: Setup node
-        uses: actions/setup-node@v4
-        with:
-          node-version: '18'
-          cache: yarn
-          cache-dependency-path: "bindings/nodejs/yarn.lock"
-
-      - name: Corepack
-        run: corepack enable
-      - name: Install dependencies
-        run: yarn install --immutable
-
-      - name: Check format
-        run: yarn run prettier --check .
-
-      - name: Build
-        run: yarn build:debug
-
-      - name: Check diff
-        run: git diff --exit-code
-
-      - name: Test bindings
-        run: yarn test
```

Comment:
This workflow is still needed since we need to check, build and running unit tests.

Reply:
Updated. Since the unit test exists in the rust code, the `cargo nextest run` is used.